src/test/resources/features/flush.feature

```diff
@@ -42,3 +44,9 @@ Feature: Flush
 		When I enter "sdk flush temp"
 		Then no "res-1.2.0.zip" file is present in temporary storage
 		And I see "1 archive(s) flushed"
+
+	Scenario: Clear out the metadata
+		Given a headers file "grails-1.3.9.headers" in metadata directory
+		When I enter "sdk flush metadata"
+		Then no metadata is cached
+		And I see "1 archive(s) flushed"	
```

Comment:
Should this read something like "Metadata was flushed"?

Reply:
This option is calling the same function (`__sdkman_cleanup_folder`) as the other option of the flush command (archives, tmp) so the message logged is the same

Not sure if I want to implement a new function